articles/virtual-desktop/store-fslogix-profile.md

```diff
@@ -11,11 +11,11 @@ ms.author: helohr
 ---
 # Storage options for FSLogix profile containers in Windows Virtual Desktop
 
-Azure offers multiple storage solutions that you can use to store your FSLogix profile container. This article compares storage solutions Azure Storage offers for Windows Virtual Desktop FSLogix profile container user profiles.
+Azure offers multiple storage solutions that you can use to store your FSLogix profile container. This article compares storage solutions that Azure offers for Windows Virtual Desktop FSLogix user profile container.
```

Comment:
This article compares storage solutions that Azure offers for Windows Virtual Desktop FSLogix user profile containers.

(Please make the last "containers" plural.)